### PR TITLE
Implemented a README.md for the report folder

### PR DESCRIPTION
**Pull request by ebradley12**
This pull request addresses #80 adds a comprehensive README file for the `report` directory, detailing the functionality, configurations, and output of the report generation script (`report_generation.py`). The README provides clear instructions for running the script, setting up environment variables, and understanding the features and outputs of the application.

### CODE REVIEW

**Review by emily-curtis — APPROVED** [review]
Great job 🤩

File: reports/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+## Configurations
+
+Ensure the following environment variables are set up:
+
+### Database Credentials:
+- `DB_HOST`
+- `DB_PORT`
+- `DB_USER`
+- `DB_PASSWORD`
+- `DB_NAME`
```
Comment (GabrielNsiah) [inline reports/README.md:61]: Maybe a description of what they correspond to in some detail here would help?
```suggestion
- `DB_HOST` - The host address of the database.
- `DB_PORT` - The port used to access the database.
- `DB_USER` - The user associated with the database.
- `DB_PASSWORD` -  The password for accessing the database.
- `DB_NAME` - The name of the database being accessed.
```

Comment (ebradley12) [reply reports/README.md:109]: Yep -  will do this

**Review by lukieh2014 — APPROVED** [review]
looks great!

**Pull request merged**